Headers/DebugServer2/Core/BreakpointManager.h

```diff
@@ -109,3 +111,5 @@ class BreakpointManager {
   virtual bool fillStopInfo(Target::Thread *thread, StopInfo &stopInfo) = 0;
 };
 } // namespace ds2
+
+ENABLE_BITMASK_OPERATORS(ds2::BreakpointManager::Lifetime);
```

Comment:
I'd put this right after line 26, so everything regarding the `Lifetime` type will be grouped together.

Reply:
The namespacing goes awry. The individual operator definitions aren't told to look into any particular namespace and so the definition of the individual `EnableBitMaskOperators` struct needs to match be defined in terms of the fully qualified typename e.g. `ds2::Something::Other::Thing`.

At least that's what I've gathered from the completely opaque error messages.